[BUG] Error related to @angular-devkit\core when using ng-morph with Angular v16

# 🐞 Bug report

### Description

I'm trying to add the [https://github.com/ng-doc/ng-doc](https://github.com/ng-doc/ng-doc) library schematics in my Angular v16 project, and this library depends on ng-morph. 
One of the processes done by ng-morph triggers the following error in my project:

```bash
Cannot find module 'D:\Lab\my-project\node_modules\@angular-devkit\core\src\virtual-fs\host.js'
```

It works correctly when tested on an Angular v15 project, so I suspect the source of the error comes from the new `esbuild` architecture used in Angular v16, [as stated here in the docs](https://angular.io/guide/update-to-version-16#faster-builds-with-the-esbuild-developer-preview). 

The path `@angular-devkit\core\src\virtual-fs\host` is indeed a **directory**, not a JS file. Ng-morph uses the shorthand syntax for imports in this file for example: [ng-morph-tree.ts#L2](https://github.com/Tinkoff/ng-morph/blob/0f855570e0cbe6f4c701f1e4c599ebc147ea4168/libs/ng-morph/project/classes/ng-morph-tree.ts#L2)

### Reproduction

```bash
# Create new Angular v16 project
npm install @angular/cli@16
ng new my-project --defaults

# Add third-party library
ng add @ng-doc/add

# Triggers error: Cannot find module 'path\to\my-project\node_modules\@angular-devkit\core\src\virtual-fs\host.js'
```

### Expected behavior

I expect no error when installing a third-party library depending on ng-morph on Angular v16.

### Versions

- Package 2.2.4

If needed:

- OS: Windows 10 & MacOS Ventura 13.4
- Angular 16.6.1 (does not work) / 15.2.0 (works as expected)

--- a/libs/ng-morph/project/classes/ng-morph-tree.spec.ts
+++ b/libs/ng-morph/project/classes/ng-morph-tree.spec.ts
@@ -25,7 +25,7 @@ describe('NgMorphTree', () => {
   it('should read from fs', () => {
     expect(getSourceFile('/ng-morph-tree.ts')?.getFullText())
       .toStrictEqual(`import { HostSink, HostTree } from '@angular-devkit/schematics';
-import { ScopedHost } from '@angular-devkit/core/src/virtual-fs/host';
+import { ScopedHost } from '@angular-devkit/core/src/virtual-fs/host/scoped';
 import { NodeJsSyncHost } from '@angular-devkit/core/node';
 import { normalize } from '@angular-devkit/core';
 

--- a/libs/ng-morph/project/classes/ng-morph-tree.ts
+++ b/libs/ng-morph/project/classes/ng-morph-tree.ts
@@ -1,5 +1,5 @@
 import { HostSink, HostTree } from '@angular-devkit/schematics';
-import { ScopedHost } from '@angular-devkit/core/src/virtual-fs/host';
+import { ScopedHost } from '@angular-devkit/core/src/virtual-fs/host/scoped';
 import { NodeJsSyncHost } from '@angular-devkit/core/node';
 import { normalize } from '@angular-devkit/core';
 

